content/news/2023-01-beacon-integration/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: 'Simple and secure publishing of variance data from Galaxy to the Beacon Network'
+date: "2023-01-24"
+tease: "Galaxy has gained a Beacon integration that enables users to conveniently publish variance data via the Beacon Network"
```

Comment:
via the Beacon Network

Which Beacon network? `beacon-network.org` is deprecated. ELIXIR has <https://beacon-network.elixir-europe.org> as v1 instance; however, v1 should not be used going forward. The correct formulation would be "... via Beacon aggregators for federated data discovery approaches."

Reply:
How would you formulate it if Galaxy is attached to one Beacon instance, and we avoid this confusing about the different networks.

Reply:
My formulation above (& below) basically leaves this open: When Beacon, then possibility to use via aggregators, in networks etc. (Network is a bit misleading use here since it is a hub + nodes topology - but people are used to this `¯\_(ツ)_/¯`).